amoro-ams/dist/src/main/amoro-bin/conf/config.yaml

```
@@ -117,6 +121,13 @@ ams:
   terminal:
     backend: local
     local.spark.sql.iceberg.handle-timestamp-without-timezone: false
+    result:
+      limit: 1000
+    stop-on-error: false
+    session:
+      timeout: 30
+    local:
+      using-session-catalog-for-hive: false
```

```suggestion
    local:
      using-session-catalog-for-hive: false
      spark.sql.iceberg.handle-timestamp-without-timezone: false
```

We may group local terminal configuration together here.